src/govuk/components/modal-dialogue/modal-dialogue.yaml

```diff
@@ -0,0 +1,120 @@
+params:
+- name: id
+  type: string
+  required: true
+  description: The id of the modal dialogue.
+- name: open
+  type: boolean
+  required: true
+  description: If true, modal dialogue element will be visible.
```

Comment:
Should we consider calling the component "modal dialog" instead? Most of the industry docs talk about "modal dialog" or "dialog element":
https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

Reply:
https://trends.google.com/trends/explore?cat=422&q=modal,dialog,modal%20dialog,modal%20dialogue,dialogue